README.md

```diff
@@ -1124,6 +1128,135 @@ Available aggregations are:
 - geometry_geohash_grid_frequency ([geohash grid](https://opensearch.org/docs/latest/aggregations/bucket/geohash-grid/) on Item.geometry)
 - geometry_geotile_grid_frequency ([geotile grid](https://opensearch.org/docs/latest/aggregations/bucket/geotile-grid/) on Item.geometry)
 
+## Asset Proxy
+
+The Asset Proxy feature enables stac-server to proxy access to S3 assets through the STAC
+API by generating requester-pays pre-signed URLs. Only assets with S3 URIs (`s3://`
+prefix) are proxied; other URL schemes are ignored. When the Asset Proxy feature is
+enabled, asset `href` values pointing to S3 are replaced with proxy endpoint URLs when an
+Item or Collection is served, while the original S3 URLs are preserved in the
+`alternate.s3.href` field using the [Alternate Assets
+Extension](https://github.com/stac-extensions/alternate-assets). Subsequent GET requests
+to the proxy endpoint URLs are redirected to pre-signed S3 object URLS for download (the
+requester pays for S3 egress).
```

Comment:
as above, this is misleading.
The proxy URL should work regardless of if the bucket is configured as requester pays or not.   In fact, it actually turns `requester pays` into `stac-server pays`.
The proxy URL is generating a pre-signed URL based on the stac-server's access, so if it was a requester pays bucket the pre-signed URL would have the egress costs assigned to the credentials used to create it.

In general, this feature should not be combined with a requester pays bucket as it defeats the purpose of that setting.